cypress/e2e/with-users/settings/configuration/general.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { generateMAASURL } from "../../../utils";
+
+context("Settings Deploy configuration", () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit(generateMAASURL("/settings/configuration/general"));
+
+    cy.findByRole("radio", { name: "Default" }).click();
+
+    cy.findByRole("button", { name: "Save" }).then(($btn) => {
+      if ($btn.is(":disabled")) {
+        return;
+      } else {
+        cy.wrap($btn).click();
+      }
+    });
```

Comment:
What's the reason for doing this before each test, shouldn't the default theme be selected by default? It would be a good idea to clean up after each test though (and revert to default theme).

Reply:
So in my instance I'm using bolla for my tests, and the theme on that is subject to change - even with a local maas instance, the theme is also subject to change if I were to leave it set to something else while manually testing. This just makes sure that before each test the theme is 100% definitely set to default, and the `if`/`else` is there to prevent any errors if the theme is already set to default